egs2/TEMPLATE/spk1/spk.sh

```diff
@@ -191,6 +191,9 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
         if [ "${skip_train}" = false ]; then
             utils/copy_data_dir.sh --validate_opts --non-print data/"${train_set}" "${data_feats}/${train_set}"
 
+            # category2utt will be used bydata sampler
```

Comment:
Can you add a documentation later in `egs2/TEMPLATE/spk1/README.md` like other tasks?
> e.g., https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/enh1/README.md

Reply:
Nice comment! I will :)